<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchQueryWidget</class>
 <widget class="QWidget" name="SearchQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelDefaultQuery">
       <property name="text">
        <string>بحث عن:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FilterLineEdit" name="lineDefaultQuery"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkDefaultQuery">
       <property name="toolTip">
        <string>كل العبارات</string>
       </property>
       <property name="text">
        <string>الكل</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widgetAdvancedSearch" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>بحث عن كل الكلمات:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="FilterLineEdit" name="lineAllWordsQuery"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>بحث مطابق:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="FilterLineEdit" name="lineExactQuery"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>بحث عن أي من الكلمات:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="FilterLineEdit" name="lineAnyWordQuery"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>استبعاد الكلمات:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="FilterLineEdit" name="lineWithoutQuery"/>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinExactQuerySlop">
          <property name="toolTip">
           <string>التباعد بين الكلمات</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkWithoutQuery">
          <property name="toolTip">
           <string>كل العبارات</string>
          </property>
          <property name="text">
           <string>الكل</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="toolAdvancedSearch">
       <property name="text">
        <string>+</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>بحث متقدم</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
